src/app/view/components/displayTimerText.js

```diff
@@ -0,0 +1,22 @@
+import elementCreator from '../../utils/elementCreator';
+
+// const tylko do testów, docelowo time powinien być pobierany z zewnątrz
+const time = 45;
+
+export default function displayTimerText() {
```

Comment:
this function now hasn't any parameter, change this function to one-parameter function displayTimerText(timeInSeconds){ }